UPGRADE-4.4.md

```diff
@@ -151,6 +151,52 @@ TwigBridge
 
  * Deprecated to pass `$rootDir` and `$fileLinkFormatter` as 5th and 6th argument respectively to the
    `DebugCommand::__construct()` method, swap the variables position.
+   
+TwigBundle
+----------
+
+ * Deprecated default value `twig.controller.exception::showAction` of the `twig.exception_controller` configuration option, set it to `null` instead
```

Comment:
This will change the default error response format. Please explain the difference with a before and after comparison.

Reply:
Done.